src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Manifest.component.tsx

```diff
@@ -415,7 +444,10 @@ function ManifestComponent({
                                                 ? EA_MANIFEST_SECRET_EDIT_MODE_INFO_TEXT
                                                 : EA_MANIFEST_SECRET_INFO_TEXT
                                         }
-                                    />
+                                        className="flex left"
+                                    >
+                                        {!isEditmode && secretViewAccess && renderShowDecodedValueCheckbox(trimedManifestEditorData)}
```

Comment:
can we use `trimedManifestEditorData` in `onChangeToggleShowDecodedValue` directly instead of passing here?